portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase

```diff
@@ -388,6 +388,36 @@ definition {
 		AssertHTMLSourceTextPresent(value1 = "${escapedScript}");
 	}
 
+	@description = "This is for LPS-117983."
+	@priority = "4"
+	test ViewStructureName {
+		task ("Create a structure named <script>alert(123)</script>") {
+			Navigator.openURL();
+
+			ProductMenu.gotoPortlet(
+				category = "Content &amp; Data",
+				portlet = "Web Content");
+
+			NavItem.gotoStructures();
+
+			WebContentStructures.addCP(structureName = '''<script>alert(123);</script>''');
+		}
+
+		task ("Add a Text field") {
+			DataEngine.addField(
+				fieldFieldLabel = "Text",
+				fieldName = "Text");
+		}
+
+		task ("Save the Structure") {
+			WebContentStructures.saveCP();
+		}
+
+		task ("View the Structure name") {
+			WebContentStructures.viewCP(structureName = '''<script>alert(123);</script>''');
+		}
+	}
```

Comment:
Add step to add web content based on new structure

Reply:
Hello @Tim-Cao,
I couldn't Add Web Content based on new Structure called `<script>alert(123);</script>`. The structure appears to be added as  `&lt;script&gt;alert(123);&lt;/script&gt;`

Reply:
No need to type name. The case test no alert popup when create a web content based on structure with XSS name.